jmxtrans-core/src/main/java/com/googlecode/jmxtrans/jmx/JmxResultProcessor.java

```diff
@@ -111,8 +111,16 @@ private void getResult(Builder<Result> accumulator, String attributeName, Compos
 		Map<String, Object> values = newHashMap();
 
 		Set<String> keys = t.keySet();
+		keyLoop:
```

Comment:
A label really hurt my eyes. Unless you can demonstrate this is the cleanest way to implement this, I'll have trouble merging it. (Sorry)